lib/foreman_remote_execution_core/script_runner.rb

```diff
@@ -44,6 +44,14 @@ def reset
     end
   end
 
+  class DzdoUserMethod < SudoUserMethod
+    LOGIN_PROMPT = /password/i
```

Comment:
This actually doesn't get used. Converting this from constant to method should help: I guess the penalty of creating a regex object every time vs. having that in constant is acceptable.